src/Shopsys/ShopBundle/Resources/translations/messages.de.po

```diff
@@ -2,884 +2,868 @@ msgid ""
-msgstr "(de) Your cart is unfortunately empty. To create order, you have to <a href=\"%url%\">choose</a> some product first"
+msgstr "Dein Warenkorb ist leider leer. Um eine Bestellung zu erstellen, müssen Sie zuerst ein Produkt <a href=\"%url%\">auswählen</a>"
 
 msgid "Your data had been successfully updated"
-msgstr "(de) Your data had been successfully updated"
+msgstr "Ihre Daten wurden erfolgreich aktualisiert"
 
 msgid "Your discount"
-msgstr "(de) Your discount"
+msgstr "Ihr Rabatt"
 
 msgid "Zasilkovna.cz - dispensing point"
-msgstr "(de) Zasilkovna.cz - dispensing point"
+msgstr "Zasilkovna.cz - Ausgabestelle"
 
 msgid "Zásilkovna"
-msgstr "(de) Zásilkovna"
+msgstr "Zásilkovna"
 
 msgid "customers"
-msgstr "(de) customers"
+msgstr "Kunden"
 
 msgid "etc. Praha"
-msgstr "(de) etc. Praha"
+msgstr "etc. Prag"
 
 msgid "excluding VAT"
-msgstr "(de) excluding VAT"
+msgstr "ohne Mehrwertsteuer"
 
 msgid "from"
-msgstr "(de) from"
+msgstr "von"
 
 msgid "from %price%"
-msgstr "(de) from %price%"
+msgstr "ab %price% Preis"
 
 msgid "on homepage above popular categories (Primary)"
-msgstr "(de) on homepage above popular categories (Primary)"
+msgstr "auf der Homepage über den populären Kategorien (Primär)"
 
 msgid "on homepage above popular categories (Secondary First)"
-msgstr "(de) on homepage above popular categories (Secondary First)"
+msgstr "auf der Homepage über den populären Kategorien (Sekundäre Zuerst)"
 
 msgid "on homepage above popular categories (Secondary Second)"
-msgstr "(de) on homepage above popular categories (Secondary Second)"
+msgstr "auf der Homepage über den populären Kategorien (Sekundäre Zweiten)"
 
 msgid "products"
-msgstr "(de) products"
+msgstr "Produkte"
 
 msgid "records"
-msgstr "(de) records"
+msgstr "Datensätze"
 
 msgid "unregistered customer"
-msgstr "(de) unregistered customer"
+msgstr "nicht registrierter Kunde"
 
 msgid "{0} No product found | {1} 1 product found | [2,Inf] %totalProductCount% product found"
-msgstr "(de) {0} No product found | {1} 1 product found | [2,Inf] %totalProductCount% product found"
+msgstr "{0} Kein Produkt gefunden | {1} 1 Produkt gefunden | [2,Inf] %totalProductCount% Produkt gefunden"
 
 msgid "{1} <strong class=\"cart__state\">%itemsCount%</strong> item for <strong class=\"cart__state\">%priceWithVat%</strong>|[2,Inf] <strong class=\"cart__state\">%itemsCount%</strong> items for <strong class=\"cart__state\">%priceWithVat%</strong>"
-msgstr "(de) {1} <strong class=\"cart__state\">%itemsCount%</strong> item for <strong class=\"cart__state\">%priceWithVat%</strong>|[2,Inf] <strong class=\"cart__state\">%itemsCount%</strong> items for <strong class=\"cart__state\">%priceWithVat%</strong>"
+msgstr "{1} <strong class=\"cart__state\">%itemsCount%</strong> Artikel für <strong class=\"cart__state\">%priceWithVat%|</strong>[2,Inf] <strong class=\"cart__state\">%itemsCount%</strong> Artikel für <strong class=\"cart__state\">%priceWithVat%</strong>"
```

Comment:
There was a mistake in `transchoice` syntax message which is why multiple cart were unable to be added to cart.

```suggestion
msgstr "{1} <strong class=\"cart__state\">%itemsCount%</strong> Artikel für <strong class=\"cart__state\">%priceWithVat%</strong>|[2,Inf] <strong class=\"cart__state\">%itemsCount%</strong> Artikel für <strong class=\"cart__state\">%priceWithVat%</strong>"
```